### PR TITLE
Add resources for SonarQube and ChartMuseum.

### PR DESCRIPTION
**Pull request by headcr4sh**

### CODE REVIEW

File: cathive-chartmuseum-resource.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: chartmuseum
+repo: https://github.com/cathive/cathive-chartmuseum-resource
```
Comment (YoussB) [inline cathive-chartmuseum-resource.yml]: The repo doesn't seem to exist, did you mean: https://github.com/cathive/concourse-chartmuseum-resource

File: cathive-sonarqube-resource.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: sonarqube
+repo: https://github.com/cathive/cathive-sonarqube-resource
```
Comment (YoussB) [inline cathive-sonarqube-resource.yaml]: Same, did you mean: https://github.com/cathive/concourse-sonarqube-resource

**Review by YoussB — CHANGES_REQUESTED** [review]
Thanks @headcr4sh for the contribution.
I think the github repo links are incorrect.
Also, all the other resources don't have the `---` in the beginning of the file or the `...` towards the end.
Since these are optional, and would be used in case of multiple yaml objects in the same file, I would rather have all the files follow the same convention.

File: cathive-sonarqube-resource.yaml (hunk 2 above)
Comment (headcr4sh) [reply cathive-sonarqube-resource.yaml]: yes. Oooops.

File: cathive-chartmuseum-resource.yml (hunk 1 above)
Comment (headcr4sh) [reply cathive-chartmuseum-resource.yml]: Yes. Damn.

**Comment by headcr4sh** [comment]
> Thanks @headcr4sh for the contribution.
> I think the github repo links are incorrect.
> Also, all the other resources don't have the `---` in the beginning of the file or the `...` towards the end.
> Since these are optional, and would be used in case of multiple yaml objects in the same file, I would rather have all the files follow the same convention.

Ok. Removed the stream separators.

**Review by YoussB — APPROVED** [review]
Perfect, Thanks @headcr4sh

**Pull request merged**